modules/kubernetes/pages/k8s-operator/connect-to-cluster.adoc

```diff
@@ -0,0 +1,38 @@
+= Connecting to a Cluster
+:description: Instructions for connecting to a cluster created by the TigerGraph Kubernetes Operator.
+
+This page describes the steps to connect to a TigerGraph cluster created by the TigerGraph Kubernetes Operator.
+
+== Establish an SSH connection to the cluster
+
+To gain shell access to your cluster, run the following command:
+
+[.wrap,console]
+----
+$ kubrctl tg connect --cluster-name <cluster_name> --namespace <namespace_name>
+----
+
+== Access GraphStudio
+To access GraphStudio, make sure port 14240 on your cluster is accessible.
+Run the following command to retrieve the IP address of the GUI service.
+This is the service that powers GraphStudio.
+
+[.wrap,console]
+----
+$ kubectl get services --namespace <namespace_name>
+----
+
+In the output, under the external-IP column, you can find the external IP of the service.
+Visit port 14240 the IP to access GraphStudio.
```

Comment:
Visit port 14240 on the external IP to access GraphStudio

Do we need a link to the graphstudio documentation?